tests/CMakeLists.txt

```diff
@@ -525,12 +525,6 @@ set_tests_properties(aktualizr_secondary_cmdline--something
                      PROPERTIES WILL_FAIL TRUE)
 
 # Check for expected output messages given specific misconfigurations
-# test the error message when running the executable with non-existent configuration file
-add_test(NAME aktualizr_secondary_no_config_check_message
-         COMMAND aktualizr-secondary -c non-existent-config.toml)
-set_tests_properties(aktualizr_secondary_no_config_check_message
-                     PROPERTIES PASS_REGULAR_EXPRESSION "aktualizr-secondary: configuration file .* not found. Exiting.")
```

Comment:
Isn't this test still valid? When the user explicitly specifies a config file that isn't present, we should tell them :)